dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -725,6 +725,17 @@
         #   port = 8000
         #   limit = 1024
 
+      # Skip records Exporter ------------
+      #
+      # Enable the exporter to skip record position
+      #
+      # These setting can also be overridden using the environment variables "ZEEBE_BROKER_EXPORTERS_[exporter name]_SKIPRECORDS"
+      # Allows to skip certain records by their position. This is useful for debugging or skipping a record that is
+      # preventing processing or exporting to continue.
+      # Record positions defined to skip in one of the exporters will be skipped in all exporters.
+      # The value is a comma-separated list of records ids to skip. Whitespace is ignored.
+      # skipRecords:
+
```

Comment:
:x: This is unexpected behavior and a bit confusing on multiple levels.
I think we should just add the `skipRecords` configuration on another level, so not in `zeebe.broker.exporters` but something like `zeebe.broker.exporting`.